src/main/java/nl/tudelft/dnainator/ui/DNAinator.java

```diff
@@ -24,20 +32,55 @@
 	private static final int MIN_WIDTH = 300;
 	private static final int MIN_HEIGHT = 150;
 
+	private BooleanProperty ready = new SimpleBooleanProperty(false);
```

Comment:
What advantage has setting the property name over just giving the instance a name?